src/main/java/shef/data/LiveStream.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shef.data;
+
+import java.util.Date;
+ 
+/** A YouTube live stream.*/
+public class LiveStream {
+  long userId;
+  long recipeKey;
```

Comment:
The recipe's Datastore keys are saved as strings. Do you mean to use String here?

Reply:
Oopsies yes i fix